app/src/main/java/com/omkarmoghe/pokemap/views/map/MapWrapperFragment.java

```diff
@@ -130,41 +127,52 @@ public void onLocationChanged(Location location) {
         locationFab.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View view) {
-                if (mLocation != null && mGoogleMap != null) {
+
+                Context context = getContext();
+
+                if (mLocation != null && mGoogleMap != null && context != null) {
                     mGoogleMap.animateCamera(CameraUpdateFactory.newLatLngZoom(
                             new LatLng(mLocation.getLatitude(), mLocation.getLongitude()), 15));
 
-                    MainActivity.toast.setText("Found you!");
-                    MainActivity.toast.show();
+                    Toast.makeText(context, "Found you!", Toast.LENGTH_SHORT).show();
```

Comment:
This was there before -- don't want to touch it because it's not related to stability. We can and should fix it in another PR (?)